src/components/week/calendarWeekView.component.ts

```diff
@@ -279,7 +284,14 @@ export class CalendarWeekViewComponent implements OnChanges, OnInit, OnDestroy {
    * @hidden
    */
   getDayColumnWidth(eventRowContainer: HTMLElement): number {
-    return Math.floor(eventRowContainer.offsetWidth / 7);
+    return Math.floor(eventRowContainer.offsetWidth / this.count());
+  }
+
+  /**
+   * @hidden
+   */
+  count(): number {
+    return 7 - this.excludeDays.length;
   }
```

Comment:
Again, just pre-calculate this once in the ngOnChanges lifecycle hook when `excludeDays ` changes.